forgerock-auth-ui/src/main/java/org/forgerock/android/auth/ui/callback/DeviceBindingCallbackFragment.kt

```diff
@@ -0,0 +1,49 @@
+/*
+ * Copyright (c) 2022 ForgeRock. All rights reserved.
+ *
+ *  This software may be modified and distributed under the terms
+ *  of the MIT license. See the LICENSE file for details.
+ */
+package org.forgerock.android.auth.ui.callback
+
+import android.os.Bundle
+import android.view.LayoutInflater
+import android.view.View
+import android.view.ViewGroup
+import kotlinx.coroutines.CancellationException
+import kotlinx.coroutines.CoroutineScope
+import kotlinx.coroutines.Dispatchers
+import kotlinx.coroutines.launch
+import org.forgerock.android.auth.callback.DeviceBindingCallback
+import org.forgerock.android.auth.ui.databinding.FragmentDeviceBindingCallbackBinding
+
+/**
+ * A simple [Fragment] subclass.
+ */
+class DeviceBindingCallbackFragment : CallbackFragment<DeviceBindingCallback?>() {
```

Comment:
DeviceBindingCallback? - does this needs to be optional

Reply:
Remove optional